sphinx_sitemap/__init__.py

```diff
@@ -28,6 +28,12 @@ def setup(app):
         rebuild=False
     )
 
+    app.add_config_value(
+        'sitemap_name',
```

Comment:
I'd prefer `sitemap_filename` as I think it is a little more self-explanatory when read in **conf.py**.
```suggestion
        'sitemap_filename',
```